packages/adders/storybook/index.ts

```diff
@@ -2,16 +2,9 @@ import { defineAdder } from '@sveltejs/cli-core';
 
 export default defineAdder({
 	id: 'storybook',
-	environments: { kit: true, svelte: true },
 	homepage: 'https://storybook.js.org',
 	options: {},
-	packages: [],
-	scripts: [
-		{
-			description: 'applies storybook',
-			args: ['storybook@8.3.6', 'init', '--skip-install', '--no-dev'],
-			stdio: 'inherit'
-		}
-	],
-	files: []
+	run: async ({ sv }) => {
+		await sv.execute(['storybook@8.4', 'init', '--skip-install', '--no-dev'], 'inherit');
```

Comment:
it seems funny to halfway pin it like this. I'd remove the `@8.4` so that folks get the latest when 8.5 comes out